packages/platform-android/src/link/patches/applyParams.ts

```diff
@@ -14,10 +14,11 @@ export default function applyParams(
   params: ProjectParamsAndroid,
   prefix: string,
 ) {
+  // @ts-ignore String.replace function must return string, not string | null.
   return str.replace(/\$\{(\w+)\}/g, (_pattern: string, param: string) => {
     const name = `${toCamelCase(prefix)}_${param}`;
 
     // @ts-ignore
-    return params[param] ? `getResources().getString(R.string.${name})` : '';
+    return params[param] ? `getResources().getString(R.string.${name})` : null;
```

Comment:
Looks like returning `'null'` string would do here?

Reply:
According to tests yes :P